packages/flutter_tools/test/integration.shard/flutter_build_apk_verbose_test.dart

```diff
@@ -49,7 +49,7 @@ void main() {
         '-v',
       ], workingDirectory: exampleAppDir.path);
       expect(
-          result.stdout, contains(RegExp(r'executing:\s+\S+gen_snapshot\s+')));
+          result.stdout, contains(RegExp(r'executing:\s+[\S ]+gen_snapshot\s+')));
```

Comment:
I think this is basically equivalent to `.+`

Reply:
Updated.